### PR TITLE
Enhance the Examples Documentation

### PR DESCRIPTION
**Pull request by larry-svds**
I changed the Example docs to work without change for vagrant users, under the assumption that if you have spun up a cluster you can make the necessary changes. 

Also in the base examples directory I wrote  more rambling discussion of all the steps you need to take that people new to these various tools can find have starting points.  (everyone is new to at least one of these technologies).  

You can see the docs easily here https://github.com/larry-svds/microservices-infrastructure/tree/fix-examples

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,9 @@ ssl/serial*
 
 # downloaded by playbook
 library/lvg.py
+
+#mac
+.DS_Store
+
+#intellij
+.idea
```
Comment (BrianHicks) [inline .gitignore]: we don't want to include any OS- or editor-specific ignores in our .gitignore, this should be only for the project. These should go in your [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

**Comment by BrianHicks** [comment]
Very nice! There are some good bones here, just need a little copy editing. If you remove the `.gitignore` stuff, I'll go ahead and merge this and then make the changes we need. I think we'll take this opportunity to deprecate the Minecraft example (it won't work on anything but DigitalOcean and Vagrant with our current setup.) Thank you for your work on this, updated documentation is always a great PR to get. :+1:

**Comment by larry-svds** [comment]
revered .gitignore changes that were mac and intellij specific.

**Comment by larry-svds** [comment]
oops.  pressed the wrong button.

**Comment by siddharthist** [comment]
Merging as @BrianHicks suggested. It does look good!

**Pull request merged**